### DISCUSSION TITLE
Money Filter Docs

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/docs#373] [Ideas] [answered]
It would be great if the docs elaborated on the use of the "money" filter.

For example you can output the value and currency doing:

```
entry.price|money('currency')
```

https://craftcms.com/docs/4.x/dev/filters.html#money
https://craftcms.com/docs/4.x/money-fields.html

**Comment by mattstein** [comment, marked as answer]
The `|money` filter always outputs a value and currency, and in this case you’re passing `'currency'` as a _formatLocale_ argument that’s documented [here](https://craftcms.com/docs/4.x/dev/filters.html#money). Since that’s not a valid locale, the default will be used—and you should see the same result using `|money()` or `|money('foo')`.

You may want the [`currency` filter](https://craftcms.com/docs/4.x/dev/filters.html#currency), instead!
